conceptnet5/vectors/query.py

```diff
@@ -113,7 +114,48 @@ def passes_filter(label, filter):
         else:
             return field_match(label, filter)
 
-    def expand_terms(self, terms, limit_per_term=10, include_neighbors=True):
+    @staticmethod
+    def _englishify(term):
+        splits = split_uri(term)
+        if len(splits) > 2:
+            englishified = '/c/en/' + splits[2]
+            return englishified
+
+    def _find_neighbors(self, term, limit_per_term, weight):
+        neighbors = []
+        for edge in self.finder.lookup(term, limit=limit_per_term):
+            if field_match(edge['start']['term'], term) and not field_match(
```

Comment:
I bet the code around here could be broken into lines less confusingly, but I'm not so worried about it because running 'black' on the file is a thing that I could just do afterward